proxy/proxy.go

```diff
@@ -3,18 +3,27 @@ package main
 import (
 	"crypto/tls"
 	"crypto/x509"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"net/http"
+	"os"
 	"strings"
 
 	"github.com/streadway/amqp"
 
-	"github.com/mishas/prometheus_amqp_proxy/proxy/config"
 	"github.com/mishas/prometheus_amqp_proxy/proxy/rpc"
 )
 
+var (
+	amqpURL      = flag.String("amqp_url", "", "URL of the AMQP server to dial")
+	amqpExchange = flag.String("amqp_exchange", "", "Name of the AMQP exchange to use")
+
+	certsDir   = flag.String("certs_dir", "", "Directory of certs for TLS connection to AMQP, or empty for non-TLS connection")
```

Comment:
Maybe you should document the names of the files within the directory that are expected.